VirtoCommerce.OrderModule.Test/Workflow/workflowjsonRead.json

```diff
@@ -0,0 +1,27 @@
+{
+	"States":[
+	{
+		"Status":"Pending Approval",
+		"NextState":
+		{"Manager":["Approved","Rejected"]}
+	},
+	{
+		"Status":"Approved",
+		"NextState":
+		{"Employee":["Paid","Cancelled"]}
+	},
+	{
+		"Status":"Rejected",
+		"NextState":
+		{"Employee":["Cancelled"]}
+	},
+	{
+		"Status":"Cancelled",
+		"NextState":{}
+	},
+	{
+		"Status":"Paid",
+		"NextState":{}
+	}
+	]
+}
```

Comment:
@vinhvoniteco 
Do we need this file to include in unit test? Since, there's no place uses it.